Source/Csla/Serialization/Mobile/MobileFormatter.cs

```diff
@@ -334,7 +358,7 @@ public IMobileObject GetObject(int referenceId)
     public byte[] SerializeToByteArray(object obj)
     {
       using var buffer = new MemoryStream();
-      var formatter = new MobileFormatter(_applicationContext);
+      var formatter = new MobileFormatter(applicationContext);
```

Comment:
Note: This looks weird. We have to create a new instance of the type we are currently using to serialize the provided object. Is that necessary due to the `fields` or legacy?

Reply:
Hmm. I might be that this used to be a static method and now it is an instance method? I'll research.